tests/Unit/Collection/ItemPriceCollectionTest.php

```diff
@@ -608,4 +612,74 @@ public function testValid()
         $collection->next();
         $this->assertFalse($collection->valid());
     }
+
+    /**
+     * @covers ::resetTaxes
+     * @uses Blesta\Pricing\Collection\ItemPriceCollection::excludeTax
+     * @uses Blesta\Pricing\Collection\ItemPriceCollection::append
+     * @uses Blesta\Pricing\Collection\ItemPriceCollection::valid
+     * @uses Blesta\Pricing\Collection\ItemPriceCollection::current
+     * @uses Blesta\Pricing\Type\ItemPrice
+     * @uses Blesta\Pricing\Type\ItemPrice::resetDiscountSubtotal
+     * @uses Blesta\Pricing\Type\ItemPrice::excludeTax
+     * @uses Blesta\Pricing\Type\ItemPrice::subtotal
+     * @uses Blesta\Pricing\Type\UnitPrice::__construct
+     * @uses Blesta\Pricing\Type\UnitPrice::setPrice
+     * @uses Blesta\Pricing\Type\UnitPrice::setQty
+     * @uses Blesta\Pricing\Type\UnitPrice::setKey
+     * @uses Blesta\Pricing\Type\UnitPrice::total
+     * @uses Blesta\Pricing\Modifier\AbstractPriceModifier::type
+     */
+    public function testResetTaxes()
+    {
+        $item1 = new ItemPrice(10);
+        $item2 = new ItemPrice(10);
+
+        $collection = new ItemPriceCollection();
+        $temp_collection = clone $collection;
+        $collection->append($item1);
+        $temp_collection->append($item2);
+
+        $collection->excludeTax(TaxPrice::EXCLUSIVE);
+        $collection->resetTaxes();
+        $this->assertEquals($collection, $temp_collection);
```

Comment:
This test seems a bit strange. I imagine you would probably want to assert that the collections are different before you reset and assert that they're the same.